src/dbnode/digest/writer.go

```diff
@@ -109,3 +112,37 @@ func (w *fdWithDigestContentsWriter) WriteDigests(digests ...uint32) error {
 	}
 	return nil
 }
+
+func CloseAll(writers ...FdWithDigestWriter) error {
+	multiErr := xerrors.NewMultiError()
```

Comment:
nit: typically we just build these as `var multiErr xerrors.MultiError`

Reply:
👍